app/views/groups/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<nav class="header">
+  <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+    <li>
+    <a href="/"><i class="fa fa-navicon"></i></a>
+    </li>
+    <li class="d-flex">
+      <h2 class="heading fw-bold align-self-center">Categories</h2>
+    </li>
+    <li>
+    <%= button_to destroy_user_session_path, method: :delete, class: "logout-btn" do %>   
+      <i class="fa fa-sign-out"></i>
+    <%end%>
+    
+    </li>
+  </ul>
+  </nav>
+<%=render 'layouts/errornotice' %>
+<% if @groups.any?%>
+<% @groups.each do |group| %>
+  <div class="p-2 m-3  category-div">
+   <div class="category-info">
+    <p> <%=image_tag "#{group.icon}" ,class:"p-2 m-1 rounded img-thumbnail"%></p>
+    <ul class="p-2 m-1">
+      <li> <p class="d-flex cat_name"><%= link_to group.name, group_path(group), class:"p-2 m-2 text-muted h2" %></p></li>
+      <li> <p class="p-2 m-2 text-muted cat_date"> <%= group.created_at.strftime("%d-%b-%Y")%></p></li>
+    </ul>
+    </div>
+    <ul class="p-2 m-1">
+    <li><p class="p-2 m-1 h3"> $<%= group.money_tracks.sum(:amount)%></p></li>
+    <li>

```

Comment:
- [ ] Please follow the design guidelines in terms of `font-family` and `font-weight` here and across your application.
- [ ] I also noticed that the category's name is cut off even though there's more than enough room to contain it. Please try to fix it in order to improve your UI 👍 
### Design Guideline
![image](https://user-images.githubusercontent.com/46839548/234674616-44acb068-f31f-46b7-99ee-9448308e13b8.png)

### Your Design
![image](https://user-images.githubusercontent.com/46839548/234676055-0ee64f4a-ab20-4128-b14d-613c860df216.png)

Reply:
Hi @codepantha, I tried to find the font on google fonts, but it is not available. So I won't be able to use the exact font family in my App.